geo-types/src/coordinate.rs

```diff
@@ -1,12 +1,14 @@
-use crate::{coord, CoordNum, Point};
-
+use crate::{coord, CoordNum, Measure, NoValue, Point, ZCoord};
 #[cfg(any(feature = "approx", test))]
 use approx::{AbsDiffEq, RelativeEq, UlpsEq};
+use num_traits::Zero;
+use std::fmt::Debug;
+use std::ops::{Add, Div, Mul, Neg, Sub};
 
-/// A lightweight struct used to store coordinates on the 2-dimensional
-/// Cartesian plane.
+/// A generic struct used to store a single coordinate with optional
+/// 3D (Z) and measurement support.
 ///
-/// Unlike `Point` (which in the future may contain additional information such
+/// Unlike [Point] (which in the future may contain additional information such
```

Comment:
FYI, we can retain the code formatting _and_ link to the docs here. Which I (subjectively) think is preferable since it indicates that we're talking about the `Point` API, and not just a random abstract notion of a 'point'

```suggestion
/// Unlike [`Point`] (which in the future may contain additional information such
```